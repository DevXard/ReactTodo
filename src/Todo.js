const Todo = ({id, todo, removeTodo}) => {
    return(
         <div data-test-id="todo" >{todo} <button  onClick={() => removeTodo(id)}>X</button></div> 
    )
}

export default Todo;