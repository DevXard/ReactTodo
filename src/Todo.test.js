import { render } from "@testing-library/react";
import Todo from "./Todo";

it("render Without crushing", () => {
    render(<Todo />)
}) 

it('matches snapshot', () => {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot()
})

