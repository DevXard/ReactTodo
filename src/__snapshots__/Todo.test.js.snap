// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    data-test-id="todo"
  >
     
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
